--- a/Engine/Settings.cpp
+++ b/Engine/Settings.cpp
@@ -94,8 +94,8 @@ getDefaultOcioConfigPaths()
 #ifdef __NATRON_LINUX__
     QStringList ret;
     ret.push_back( QString::fromUtf8("/usr/share/OpenColorIO-Configs") );
-    ret.push_back( QString( binaryPath + QString::fromUtf8("../share/OpenColorIO-Configs") ) );
-    ret.push_back( QString( binaryPath + QString::fromUtf8("../Resources/OpenColorIO-Configs") ) );
+//    ret.push_back( QString( binaryPath + QString::fromUtf8("../share/OpenColorIO-Configs") ) );
+//    ret.push_back( QString( binaryPath + QString::fromUtf8("../Resources/OpenColorIO-Configs") ) );
 
     return ret;
 #elif defined(__NATRON_WIN32__)
